share/functions/__fish_preview_current_file.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_preview_current_file --description "Open the current file in a pager"
+    # set pager to user-preferred pager or less
+    set -l pager "less --"
+    set -q PAGER && echo $PAGER | read -at pager
+
+    # get current or last token, use eval to resolve $HOME and ~
+    set -l file (eval echo (commandline -t))
+    test -z "$file" && set -l file (eval echo (commandline -oc)[-1])
+
+    # strip -option= from token if necessary
+    not test -e "$file" && set -l file (eval echo (string replace -r -- '^-[^\s=]*=(.*)' '$1' "$file"))
+
+    test -f "$file" && eval $pager $file
```

Comment:
With the above change to `pager` there is no longer a need to use `eval` here. So drop it, since prevents previewing files with special characters.

Reply:
done